From 2e52aa3d4aa632fa371079c3a84088a6a3fc5465 Mon Sep 17 00:00:00 2001
From: Piyush Kumar <piyushkumg@gmail.com>
Date: Mon, 13 Apr 2020 21:58:58 +0530
Subject: [PATCH] mm: Fix-up-v2 "ZEN: INTERACTIVE: Enable background reclaim of
 hugepages"

---
 mm/huge_memory.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/mm/huge_memory.c b/mm/huge_memory.c
index 90d3741c7a7d..a1897f2a9a8c 100644
--- a/mm/huge_memory.c
+++ b/mm/huge_memory.c
@@ -54,7 +54,6 @@ unsigned long transparent_hugepage_flags __read_mostly =
 	(1<<TRANSPARENT_HUGEPAGE_REQ_MADV_FLAG)|
 #endif
 	(1<<TRANSPARENT_HUGEPAGE_DEFRAG_KSWAPD_OR_MADV_FLAG)|
-#endif
 	(1<<TRANSPARENT_HUGEPAGE_DEFRAG_KHUGEPAGED_FLAG)|
 	(1<<TRANSPARENT_HUGEPAGE_USE_ZERO_PAGE_FLAG);
 
