From 26b8363157fca7d4c150983796045a3f4da374ab Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Mon, 1 May 2017 01:00:51 +0000
Subject: [PATCH] mm: reduce vmstat wakeups

Signed-off-by: Piyush Kumar <piyushkumg@gmail.com>
---
 mm/vmstat.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/vmstat.c b/mm/vmstat.c
index f8942160fc95..86e57102c712 100644
--- a/mm/vmstat.c
+++ b/mm/vmstat.c
@@ -1799,7 +1799,7 @@ static const struct seq_operations vmstat_op = {
 
 #ifdef CONFIG_SMP
 static DEFINE_PER_CPU(struct delayed_work, vmstat_work);
-int sysctl_stat_interval __read_mostly = HZ;
+int sysctl_stat_interval __read_mostly = 8 * HZ;
 
 #ifdef CONFIG_PROC_FS
 static void refresh_vm_stats(struct work_struct *work)
