diff --git a/arch/Kconfig b/arch/Kconfig
index 23702ce..cd34508 100644
--- a/arch/Kconfig
+++ b/arch/Kconfig
@@ -655,7 +655,6 @@ config ARCH_SUPPORTS_GCC_LTO
 # This might not be the exact range with this bug.
 config BAD_AR
 	depends on LD_VERSION = 23000
-	depends on $(shell,ulimit -n) < 4000
 	def_bool y
 
 choice
@@ -710,7 +709,7 @@ config LTO_GCC
 	bool "gcc LTO"
 	depends on ARCH_SUPPORTS_GCC_LTO && CC_IS_GCC
 	depends on GCC_VERSION >= 90000
-	depends on LD_VERSION >= 22700
+	depends on LD_VERSION >= 22700 || LLD_VERSION >= 100000
 	depends on !BAD_AR
 	select LTO
 	select CC_NEEDS_FRE_WA if GCC_VERSION < 100300
