From e996a3d1560d0b05f7e558a500b64533ca0664fe Mon Sep 17 00:00:00 2001
From: Diab Neiroukh <lazerl0rd@thezest.dev>
Date: Tue, 26 Jan 2021 14:26:02 +0000
Subject: [PATCH] x86: Allow using Dead Code Elimination.

Signed-off-by: Diab Neiroukh <lazerl0rd@thezest.dev>
---
 arch/x86/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/Kconfig b/arch/x86/Kconfig
index e3c59118ab94..6a7610745977 100644
--- a/arch/x86/Kconfig
+++ b/arch/x86/Kconfig
@@ -210,6 +210,7 @@ config X86
 	select HAVE_PERF_EVENTS
 	select HAVE_PERF_EVENTS_NMI
 	select HAVE_HARDLOCKUP_DETECTOR_PERF	if PERF_EVENTS && HAVE_PERF_EVENTS_NMI
+	select HAVE_LD_DEAD_CODE_DATA_ELIMINATION
 	select HAVE_PCI
 	select HAVE_PERF_REGS
 	select HAVE_PERF_USER_STACK_DUMP
