From d5a12d169ac5db813e93cebdd1cd33e5695f5528 Mon Sep 17 00:00:00 2001
From: Sultan Alsawaf <sultan@kerneltoast.com>
Date: Sun, 13 May 2018 13:02:33 -0700
Subject: [PATCH] init: Kconfig: Don't force DEBUG_KERNEL when EXPERT is
 enabled

Signed-off-by: Sultan Alsawaf <sultan@kerneltoast.com>
Signed-off-by: Diab Neiroukh <lazerl0rd@thezest.dev>
---
 init/Kconfig | 2 --
 1 file changed, 2 deletions(-)

diff --git a/init/Kconfig b/init/Kconfig
index 46ab4a6e3a80..91576a93ba13 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -1412,8 +1412,6 @@ config BPF
 
 menuconfig EXPERT
 	bool "Configure standard kernel features (expert users)"
-	# Unhide debug options, to make the on-by-default options visible
-	select DEBUG_KERNEL
 	help
 	  This option allows certain base kernel options and settings
 	  to be disabled or tweaked. This is for specialized
